includes/wc-core-functions.php

```diff
@@ -2004,3 +2004,42 @@ function wc_cleanup_session_data() {
 	}
 }
 add_action( 'woocommerce_cleanup_sessions', 'wc_cleanup_session_data' );
+
+/**
+ * Convert a decimal (e.g. 3.5) to a fraction (e.g. 7/2).
+ * From: https://www.designedbyaturtle.co.uk/2015/converting-a-decimal-to-a-fraction-in-php/
+ *
+ * @param float $decimal the decimal number.
+ * @return array|bool a 1/2 would be [1, 2] array (this can be imploded with '/' to form a string).
+ */
+function wc_decimal_to_fraction( $decimal ) {
+	if ( 0 > $decimal || ! is_numeric( $decimal ) ) {
+		// Negative digits need to be passed in as positive numbers and prefixed as negative once the response is imploded.
+		return false;
+	}
+
+	if ( 0 === $decimal ) {
+		return [ 0, 0 ];
```

Comment:
This syntax isn't supported on old PHP versions (https://travis-ci.org/woocommerce/woocommerce/jobs/333246051)

Reply:
And it should be [ 0, 1 ]. [0/0 is undefined](https://brilliant.org/wiki/what-is-0-0/). :P